legacy/python/lang/src/pip_requirements.rs

```diff
@@ -0,0 +1,15 @@
+use cached::proc_macro::cached;
+use moon_lang::LockfileDependencyVersions;
+use rustc_hash::FxHashMap;
+use std::fs;
+use std::path::PathBuf;
+
+#[cached(result)]
+pub fn load_lockfile_dependencies(path: PathBuf) -> miette::Result<LockfileDependencyVersions> {
+    let mut deps: LockfileDependencyVersions = FxHashMap::default();
+
+    let lockfile = fs::read_to_string(path.as_path()).expect("Unable to read file");
```

Comment:
You'll need an actual `requirements.txt` parser and the https://crates.io/crates/requirements crate is too old.